CryptoLightNews/Model/CoinsModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CoinsModel.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 27/01/2023.
+//
+
+import Foundation
+
+struct CoinsModel: Codable {
```

Comment:
Models don't need to have the word `Model`. And they should be in the singular.
```suggestion
struct Coin: Codable {
```